src/main/java/org/example/CinemaEmployees.java

```diff
@@ -0,0 +1,26 @@
+package org.example;
+
+import java.util.Calendar;
+
+abstract class BaseEmployee {
+    private String firstName;
+    private String lastName;
+    protected final int employmentYear;
+    protected final double defaultBaseSalary;
```

Comment:
to powinno być baseSalary i powinieneś tu trzymać pensję domyślną, ale również i pensję managera